Add concept, fix capitalization, match link text. (#14414)
cockroachdb/docs: v22.1/multiregion-overview.md

docs_area: deploy
---

This page provides an overview of CockroachDB multi-region capabilities.

## Overview

CockroachDB multi-region capabilities make it easier to run global applications. To use these capabilities effectively, you should understand the following concepts:

- [_Cluster region_](#cluster-regions) is a geographic region that you specify at node start time.

> - [_Database region_](#database-regions) is a geographic region that a database operates within. You must choose a database region from the list of available cluster regions.

Reviewer: ```suggestion
- [_Database region_](#database-regions) is a geographic region wherein a database operates. You must choose a database region from the list of available cluster regions.
```

Reviewer: Alternatively, "in which".

> - [_Super region_](#super-regions) is a set of database regions in which contained schema objects have all of their replicas stored _only_ in regions that are members of the super region.

Author: ```suggestion
```
